### PR TITLE
DOCS-5910 Path updates for rtfctl

### PR DESCRIPTION
**Pull request by paigehargrave**

### CODE REVIEW

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 1]
```diff
@@ -18,9 +18,9 @@ Install the rtfctl utility to manage Runtime Fabrics locally, with support for t
 You may need root privileges to a controller node running Runtime Fabric to follow this procedure.
 
 . Download the https://runtime-fabric.s3.amazonaws.com/rtfctl/rtfctl-0.2.32[latest version] of the `rtfctl` utility.
-. Transfer the `rtfctl` binary to a controller node under the `/usr/local/bin` folder.
+. Transfer the `rtfctl` binary to a controller node under the `/local/usr/bin` folder.
```
Comment (sundippatel) [inline modules/ROOT/pages/install-rtfctl.adoc]: I think this should still be /usr/local/bin

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 2]
```diff
@@ -18,9 +18,9 @@ Install the rtfctl utility to manage Runtime Fabrics locally, with support for t
...
 . Rename the binary to `rtfctl`.
-. Make the script executable by running `chmod +x /usr/bin/rtfctl`.
+. Make the script executable by running `chmod +x /local/usr/bin/rtfctl`.
```
Comment (sundippatel) [inline modules/ROOT/pages/install-rtfctl.adoc]: We need this path to align to the one above.
/usr/local/bin/rtfctl

**Review by sundippatel — DISMISSED** [review]

File: modules/ROOT/pages/install-rtfctl.adoc (hunk 1 above)
Comment (paigehargrave) [reply modules/ROOT/pages/install-rtfctl.adoc]: Got it - thanks!

File: modules/ROOT/pages/install-rtfctl.adoc (hunk 2 above)
Comment (paigehargrave) [reply modules/ROOT/pages/install-rtfctl.adoc]: Got it - thanks!

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**